std/concurrency.d

```diff
@@ -74,6 +74,14 @@ import std.traits;
 
 private
 {
+    import core.atomic;
```

Comment:
Can you remove the un-needed imports? (core.thread, core.sync.mutex, core.sync.condition, std.range.primitives)